packages/x-components/src/x-modules/search/components/fallback-disclaimer.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div
+    v-if="$x.isNoResultsWithFilters"
+    class="x-fallback-disclaimer"
+    data-test="fallback-disclaimer"
+  >
+    <slot v-bind="{ query }">
```

Comment:
Should we bind the query from the alias api too?
I think that doing that the component will be more tolerant to changes, for example, if we change the script to the `defineComponent` api, this will continue working.
Something like:

```
    <slot v-bind="{ query: $x.query.search }">
      No results found for {{ $x.query.search }} with the selected filters. The filters have been
      unselected.
    </slot>
```